### ISSUE TITLE
Periodic Actions close the keyboard on android (making my app unusable)

### ISSUE THREAD

**Issue by eschloss** [FlutterFlow/flutterflow-issues#3333]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

With the newest release of flutterflow, suddenly all the pages of my app that have periodic actions have unexpected behavior with keyboards. The periodic action immediately closes an open keyboard. (tested on Pixel 6 and on various pages in my app)

It occurs all throughout my application making it unusable. Please fix this ASAP.

### Expected Behavior

When a user clicks on a text field, the keyboard should open and not close when periodic actions are triggered.

### Steps to Reproduce

I haven't tested this on a blank project.
1. add a text widget
2. start a periodic action that recurs every 1000 ms.
3. try writing something in the text widget (this must be done on an actual phone)

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFXlcmAvIV1osRE+YfuL/oxvDowGUALa+dI0tZ+TQ0jF5D4PYoDeOClQFpoTuazaH9pPmGlrD4G+vz8i9j9M/AoaBeuGKU7/6l+dRPPbEChR5DQCbisfEdAONFbI2bF5qerhx1vLfp1dEYF3XGle9iQRT2Cf9qOYwx5e6fDbOY=

### Visual documentation

I don't have time to make a GIF right now. Surely others must be reporting this too.

### Environment

```markdown
- FlutterFlow version: v4.1.65 
- Platform: mobile -- android (i haven't tested on ios yet)
- Browser name and version:
- Operating system and version affected:
```

**Comment by msusviela** [comment]
Hi @eschloss. Thanks for submitting the issue. I've taken a look at your project, but with a lot going on, it's challenging to pinpoint the issue. Given the nature of the problem you're highlighting, we need a blank project with the issue steps set up in it, or at least to know in which screens you have this behavior and the widgets involved. Without these details, it might be a bit challenging for us to investigate the issue. Please provide the required information. If the issue is already resolved for you, please feel free to close the issue. Thanks!

**Comment by eschloss** [comment]
Thank your for the quick response! 
I don't have time to set up a blank project, but I can tell you where in my project to look. 

In the meantime, I found a temporary solution that keeps the app functioning for me (not optimally, but tolerably), but it is not a solution - just a workaround.
 
**I'm more concerned about the stability of flutterflow code in general. It's possible that this is causing relatively hidden bugs in loads of other teams' projects.**

There are three separate instances that this problem occurs in my current project:
1. In the "PulsePage" Page. Look at the "OnPageLoad". With the Periodic Timer "getlocationtimer". Anytime this timer triggers (every 1000ms), it disables the keyboard. This primarily occurs with the TextField in the "PulseV2" Component.
(My temporary workaround is to have the timer deleted after its first call with StopPeriodicAction.)

2. In the "DiscoverPage" Page. Look at the "OnPageLoad". With the Periodic Timer "getlocationTimer2". Anytime this timer triggers (every 1000ms), it disables the keyboard. This occurs in a few different place. A) After a user clicks a button that navigates them to the "ChooseCity" Page. Clicking on the TextField opens up the keyboard, but within 1000ms, this keyboard disappears. B) After a user clicks on a button that navigates them to the "CreateEvent" Page. Every single TextField widget gets interrupted after clicking on them within 1000ms.
(As before, my temporary workaround is to have the timer deleted after its first call with StopPerioicAction.)

4. In the "ChatConversationList". Look at the "OnPageLoad". With the Periodic Timer "instantTimer". Anytime this timer triggers (every 1000ms), it disables the keyboard. This primarily occurs on the "ChatBetweenUsers" Page within the "Messages" component, within the "SendMessage" component --> TextField1.
(As before, my temporary fix is to have the timer deleted after its first call.)

One interesting thing to note is that the "Messages" Component has a timer that loads OnPageLoad called "refreshPeriodic" that runs every 3000ms. This one does not interrupt the keyboard on the SendMessage-->TextField1 Component.

Also, I emptied out the Periodic Timer "getlocationtimer" to test if it was caused by code within this timer. Even though the timer was empty, it still caused the keyboard to disappear every 1000ms. This should suggest that it is caused by a bug with the periodic timer. However, I can't say for certain since I'm not aware of flutterflow's underlying code.
